### PR TITLE
[DEP-290] - Build a new landing page for Covid-19 Job Retention Scheme

### PR DESCRIPTION
**Pull request by jassalrichy**

### CODE REVIEW

File: test/views/html/pages/JobRetentionSchemeViewSpec.scala [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import views.html.JobRetentionSchemeView
+
+class JobRetentionSchemeViewSpec extends ChatViewBehaviours {
+  implicit override val fakeRequest = FakeRequest("GET", "/").withCookies(Cookie("mdtp", "12345"))
+
+  val view = app.injector.instanceOf[JobRetentionSchemeView]
+
+  def createView: () => HtmlFormat.Appendable = () => view()(fakeRequest, messages)
+
+  "Vat registration view" must {
```
Comment (Burdis) [inline test/views/html/pages/JobRetentionSchemeViewSpec.scala:31]: Change to Job retention scheme view

File: test/views/html/pages/JobRetentionSchemeViewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    val returnUrl: String =
+      "https://www.gov.uk/government/organisations/hm-revenue-customs/contact/vat-registration-applications-exceptions-and-changes"
```
Comment (Burdis) [inline test/views/html/pages/JobRetentionSchemeViewSpec.scala:33]: Change to Job retention scheme URL

File: test/views/html/pages/JobRetentionSchemeViewSpec.scala (hunk 1 above)
Comment (jassalrichy) [reply test/views/html/pages/JobRetentionSchemeViewSpec.scala]: Done

File: test/views/html/pages/JobRetentionSchemeViewSpec.scala (hunk 2 above)
Comment (jassalrichy) [reply test/views/html/pages/JobRetentionSchemeViewSpec.scala]: There is no return URL for this page. URL set to '#' as required for test behaviour.

**Pull request merged**